lib/_007/Parser/Actions.pm

```diff
@@ -553,7 +553,7 @@ class _007::Parser::Actions {
     }
 
     method term:regex ($/) {
-        make Q::Term::Regex.new(:contents($<contents>.made));
+        make Q::Term::Regex.new(:contents($<contents>.ast.value));
```

Comment:
We just narrowly avoided a rant about how bad `.made` is as a method name. Good going! :beer:

About the `.value` bit: see below about `Val` vs `Q`.